components/dropdown/dropdown-menu.js

```diff
@@ -65,6 +65,15 @@ class DropdownMenu extends DropdownContentMixin(LitElement) {
 				menu.focus();
 			}, 0);
 		}
+
```

Comment:
I believe applying focus will also scroll the item into view. If so, then I think the scrolling code should be in the `else` block of the `if (this__applyFocus) {`.

Reply:
This scroll code also vertically centers the selected item in the visible space, which focus doesn't. So I can do that but it would be a minor inconsistency in the behavior.

Reply:
Ah OK, thanks. I'm good with keeping it so it's consistent!